botfront-project/templates/ingress.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
   name: {{ .Release.Name }}-rasa-ingress
   annotations:
     {{- if .Values.ingress.nginx.enabled}}
+    #uncomment the following line if you are working with multiple rasa using a websocket based channel
+    # web socket connection need to have a sticky session https://socket.io/docs/using-multiple-nodes/
+    #nginx.ingress.kubernetes.io/affinity: "cookie"
     kubernetes.io/ingress.class: nginx
     nginx.org/websocket-services: {{ .Release.Name }}-rasa-service
```

Comment:
Same as above. `ingress.nginx.socketEnabled` (default: true).
Same: doc (or table) should be mention that it should be enabled if a channel using websockets or long polling (such as Rasa Webchat) is used.

Reply:
Move it to the table for rasa values and removed the old comment